tests/rule_based_profiler/parameter_builder/test_unexpected_count_statistics_multi_batch_parameter_builder.py

```diff
@@ -760,3 +760,28 @@ def test_unexpected_count_statistics_multi_batch_parameter_builder_bobby_check_s
         "round_decimals",
         "evaluation_parameter_builder_configs",
     }
+
+
+def test_column_value_missing_data_assistant_standardize_mostly(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    # tests for UnexpectedCountStatisticsMultiBatchParameterBuilder._standardize_mostly
+    data_context: DataContext = (
+        bobby_columnar_table_multi_batch_deterministic_data_context
+    )
+
+    # noinspection PyUnusedLocal
+    builder: ParameterBuilder = UnexpectedCountStatisticsMultiBatchParameterBuilder(
+        name="my_name",
+        aggregation_method="all_values",
+        unexpected_count_parameter_builder_name="my_unexpected_count",
+        total_count_parameter_builder_name="my_total_count",
+        data_context=data_context,
+    )
+    assert builder._standardize_mostly(1.0) == 1.0
+    assert builder._standardize_mostly(0.99) == 0.99
+    assert builder._standardize_mostly(0.997) == 0.99
+    assert builder._standardize_mostly(0.98) == 0.975
+    assert builder._standardize_mostly(0.975) == 0.975
+    assert builder._standardize_mostly(0.96) == 0.96
+    assert builder._standardize_mostly(0.2) == 0.2
```

Comment:
@TrangPham I think that you are in a great position to add a test where `aggregation_method` is `"active_batch_value"`.  All my tests were for the value being `"all_values"`, because I did not have the logic for `mostly` -- but since you have added it, one test would be great (the one in the style of `_metric_dependencies_evaluated_in_parameter_builder` would be the simplest and sufficient).  Thank you!